tests/cypress/component/loading-indicator.cy.js

```diff
@@ -0,0 +1,34 @@
+import LoadingIndicator from '../../../resources/js/components/shared/LoadingIndicator.vue';
+
+it('displays according to isLoading prop', () => {
+  cy.mount(LoadingIndicator, {
+    props: {
+      isLoading: true,
+    },
+    slots: {
+      default: 'Rendered content',
+    },
+  });
+
+  // The loading indicator has a delay to prevent a flicker on pages which render immediately.
+  cy.wait(1000);
+
+  cy.get('img[alt="The page is loading."').should('be.visible');
```

Comment:
Is this missing a closing bracket?
```suggestion
  cy.get('img[alt="The page is loading."]').should('be.visible');
```
Same on line 31.